docs/src/app/components/pages/components/IconMenu/Page.jsx

```diff
@@ -13,12 +13,15 @@ import IconMenuExampleControlled from './ExampleControlled';
 import iconMenuExampleControlledCode from '!raw!./ExampleControlled';
 import IconMenuExampleScrollable from './ExampleScrollable';
 import iconMenuExampleScrollableCode from '!raw!./ExampleScrollable';
+import IconMenuExampleNested from './ExampleNestedDrop';
+import iconMenuExampleNestedCode from '!raw!./ExampleNestedDrop';
 
 const descriptions = {
   simple: 'Simple Icon Menus demonstrating some of the layouts possible using the `anchorOrigin` and `' +
   'targetOrigin` properties.',
   controlled: 'Two controlled examples, the first allowing a single selection, the second multiple selections.',
   scrollable: 'The `maxHeight` property limits the height of the menu, above which it will be scrollable.',
+  nested: 'Example of nested menus within a icon menu.',
 };
```

Comment:
I agree, I think whenever we're mentioning another component we should use the component name with back ticks.

```
`IconMenu`
```